spec/features/artists/show_spec.rb

```diff
@@ -20,7 +20,25 @@
 
     visit "/artists/#{@artist1.id}"
 
-    expect(page).to have_content(@painting1.name)
-    expect(page).to have_content(@painting1.description)
+    expect(page).to have_css(".painting", count: 1)
+
+    within(first(".painting")) do
+      expect(page).to have_css(".name")
+      name = find(".name").text
+      expect(name).to_not be_empty
+
+      expect(page).to have_css(".description")
+      description = find(".description").text
+      expect(description).to_not be_empty
+
+      expect(page).to have_css(".medium")
+      medium = find(".medium").text
+      expect(medium).to_not be_empty
+
+      expect(page).to have_css(".color_palette")
+      color_palette = find(".color_palette").text
+      expect(color_palette).to_not be_empty
+    end
```

Comment:
Since we are generating the data with a factory association (only creating artist and then automating creation of art) we won't know what the art contains. This is how we can test that those fields exist!